### PR TITLE
Update batch-add clear exit ug

### PR DESCRIPTION
**Pull request by prit3010**
Updated batch-add, clear and exit in the User Guide

Now batch-add includes clear steps from creating CSV file to running the command

### CODE REVIEW

**Comment by prit3010** [comment]
Fixes #190

File: docs/UserGuide.md [hunk 1]
```diff
@@ -192,35 +192,92 @@ Example:
...
+| Index | Field            | Mandatory Field? |
+|-------|------------------|------------------|
+| 1.    | Name             | **Yes**              |
+| 2.    | Phone            | No               |
+| 3.    | Email            | No               |
+| 4.    | Position         | **Yes**              |
+| 5.    | Department       | **Yes**              |
+| 6.    | Address          | No               |
+| 7.    | Number of Leaves | No               |
+| 8.    | Tags             | No               |
```
Comment (chao890) [inline docs/UserGuide.md:231]: Please change table to follow master branch format

File: src/main/java/coydir/logic/commands/BatchAddCommand.java [hunk 2]
```diff
@@ -60,14 +60,23 @@ public List<AddCommand> getInfo() throws CommandException {
...
                     } else {
-                        arg += PREFIX_LIST[i] + data[i] + " ";
+                        if (data[i].equals("")) {
+                            continue;
+                        } else {
+                            arg += PREFIX_LIST[i] + data[i] + " ";
+                        }
                     }
                 }
+                System.out.println(arg);
```
Comment (chao890) [inline src/main/java/coydir/logic/commands/BatchAddCommand.java:79]: Pls remove

**Review by chao890 — CHANGES_REQUESTED** [review]

File: src/main/java/coydir/logic/commands/BatchAddCommand.java (hunk 2 above)
Comment (Kok-je) [inline src/main/java/coydir/logic/commands/BatchAddCommand.java:79]: This should be deleted

**Review by Kok-je — CHANGES_REQUESTED** [review]
LTGM

**Review by Kok-je — APPROVED** [review]
LGBT

**Review by chao890 — APPROVED** [review]

**Pull request merged**